css/css-properties-values-api/at-property-shadow.html

```diff
@@ -38,9 +38,9 @@
   root.append(template.content.cloneNode(true));
   let inside = root.querySelector('#inside');
   assert_equals(getComputedStyle(outside).getPropertyValue('--x'), '2px');
-  assert_equals(getComputedStyle(outside).getPropertyValue('--y'), 'calc(1px + 1px)');
+  assert_equals(getComputedStyle(outside).getPropertyValue('--y'), '2px');
   assert_equals(getComputedStyle(inside).getPropertyValue('--x'), '2px');
-  assert_equals(getComputedStyle(inside).getPropertyValue('--y'), 'calc(1px + 1px)');
-}, '@property rules in shadow trees should have no effect');
+  assert_equals(getComputedStyle(inside).getPropertyValue('--y'), '2px');
+}, '@property rules in shadow trees should be globally registered');
```

Comment:
Does like 19 need to be removed in this case?

Reply:
Thanks! I’ve updated the stale comment